<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Class 1 Informatics | BMB 101</title>
  <meta name="description" content="<p>This is an open-source online class for new graduate students in biochemistry and molecular biophysics.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  <meta name="generator" content="bookdown 0.36 and GitBook 2.6.7" />

  <meta property="og:title" content="Class 1 Informatics | BMB 101" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="<p>This is an open-source online class for new graduate students in biochemistry and molecular biophysics.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Class 1 Informatics | BMB 101" />
  
  <meta name="twitter:description" content="<p>This is an open-source online class for new graduate students in biochemistry and molecular biophysics.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  

<meta name="author" content="Michael Greenberg, PhD" />


<meta name="date" content="2025-06-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">BMB 101</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="1" data-path="informatics.html"><a href="informatics.html"><i class="fa fa-check"></i><b>1</b> Informatics</a>
<ul>
<li class="chapter" data-level="1.1" data-path="informatics.html"><a href="informatics.html#pre-reading"><i class="fa fa-check"></i><b>1.1</b> Pre-reading</a></li>
<li class="chapter" data-level="1.2" data-path="informatics.html"><a href="informatics.html#in-class-activity"><i class="fa fa-check"></i><b>1.2</b> In-class activity</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">BMB 101</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="informatics" class="section level1 hasAnchor" number="1">
<h1><span class="header-section-number">Class 1</span> Informatics<a href="informatics.html#informatics" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="pre-reading" class="section level2 hasAnchor" number="1.1">
<h2><span class="header-section-number">1.1</span> Pre-reading<a href="informatics.html#pre-reading" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong><u>Public repositories and “Omics”</u></strong></p>
<p>As scientific knowledge has grown, there has also been a push to make
this data accessible to others in the field. Both funding agencies and
publishers have been pushing to increase public access to data in
curated public repositories. Importantly, advances in technologies
(e.g., sequencing, mass spectrometry) have enabled the production of
large “omics” data sets (e.g., such as genomic, transcriptomic,
proteomic, and metabolomic) for many different cell types, tissues, and
disease states. Moreover, open-sourced computational tools have been
developed to enable scientists to mine and draw new insights from
existing, publicly available datasets.</p>
<p>Before starting a new project (or working on a qualifying exam), it is
almost always helpful to dig into some of this data to form rigorous
hypotheses. A little bit of time spent mining some of these databases
before starting a project can potentially save months or even years of
trouble. Some questions you should ask before starting a project
include:</p>
<ul>
<li><p>Where is this molecule expressed? Is it expressed during
&gt; health/disease/development?</p></li>
<li><p>Are there multiple variants of this molecule (e.g., splice variants,
&gt; homologs)? Which variant should I study to address my specific
&gt; scientific question?</p></li>
<li><p>What is known about the sequence? Are there certain regions that are
&gt; strongly conserved? Does it share sequence homology with other
&gt; molecules that can give insights into its function or functional
&gt; domains?</p></li>
<li><p>Is this molecule associated with disease? Are there specific
&gt; pathogenic mutations that have been seen in patient populations?
&gt; Where do these mutations occur in the sequence? Are these
&gt; mutations likely pathogenic or just genetic variation?</p></li>
<li><p>Is anything known about the structure of the molecule or its
&gt; structural domains? If so, can insights be drawn from knowledge of
&gt; conservation or the location of pathogenic mutations? Are there
&gt; predicted disordered regions that might pose challenges to
&gt; conventional structural techniques?</p></li>
<li><p>What are the known binding partners? Is this molecule part of a
&gt; larger macromolecular complex? Do binding partners vary in
&gt; different cellular contexts?</p></li>
<li><p>Is the molecule modified (e.g., PTMs)? Is anything known about these
&gt; modifications and what they mean for function?</p></li>
<li><p>Is it possible to target my molecule with known small molecules?</p></li>
</ul>
<p>A lot of this information is already available in large public
repositories and knowing it can help you to form better hypotheses,
design careful and meaningful experiments, and interpret your data.</p>
<p><u>Important disclaimer:</u></p>
<p>There are many tools and repositories out there. My goal is to give a
basic workflow that you can apply to your own data. The goal is NOT to
give an exhaustive study of these tools and repositories. The goal is
NOT to dive into all of the features of some of these databases/tools,
rather we will only look at key features. Finally, there are multiple
ways to tackle these problems, and my strategy here is definitely not
the only way to do things, and it almost certainly isn’t the most
efficient way to do things. That being said, the hope is that the
approach here will be useful to you as a jumping off point.</p>
<p><u>Some publicly available databases:</u></p>
<ol style="list-style-type: decimal">
<li><p>National Center for Biotechnology Information (NCBI) – National
Library of Medicine: This is the well-curated NIH repository. It
contains MANY useful resources, including DNA/RNA sequences, protein
sequences, structure, taxonomy, lists of mutations, small molecules,
and gene expression datasets (GEO).</p></li>
<li><p>UniProt – Protein sequence, structure, and function</p></li>
<li><p>Human protein atlas – expression of genes and proteins in different
tissues, diseases, cell types, and antibodies.</p></li>
<li><p>ClinVar – Variants reported by clinicians. This includes
pathogenic, benign, and variants of unknown significance.</p></li>
<li><p>gnomAD – This contains annotated lists of “healthy” human exomes
and genomes.</p></li>
<li><p>Protein Data Bank (PDB) – molecular structures</p></li>
<li><p>Electron Microscopy Database (EMDB) – Cryo-EM structures</p></li>
<li><p>MassIVE Database – Mass spectrometry and post-translational
modifications</p></li>
<li><p>ChEMBL – A database with lists of small molecules targeting
different molecules. (Selleckchem is a company that also has great
lists of inhibitors and activators).</p></li>
<li><p>Metabolomics Workbench – Repository or metabolomics data.</p></li>
<li><p>NIH Common Fund's National Metabolomics Data Repository</p></li>
</ol>
<p><u>Some useful online computational tools:</u></p>
<ol style="list-style-type: decimal">
<li><p>BLAST – Searches known nucleotide and protein sequences against an
unknown. Useful for looking at conservation and related sequences.</p></li>
<li><p>Clustal Omega – Multiple sequence alignment</p></li>
<li><p>AlphaFold – Protein structure prediction from sequence</p></li>
<li><p>AIUPred – Prediction of intrinsically disordered proteins</p></li>
<li><p>ProtParam – Protein property predictions including isoelectric
point, molecular weight, extinction coefficient, hydrophobicity</p></li>
</ol>
<p><u>NCBI and UniProt</u></p>
<p>If you are looking for information about a gene or protein, a great
starting point is to look it up in NCBI and UniProt. Both databases
contain expertly curated information about genes and proteins. Whenever
you start a project, this is a great place to begin your studies. Both
websites contain information curated in different ways. It should be
noted that there is WAY more information on these websites than I will
cover in this short introduction, and I encourage you to spend some time
with your favorite protein.</p>
<p>UniProt assigns a unique identifier to each protein (e.g., MYH7 is
P12883). When you look up your protein of interest, you can get
information about (with linked publications):</p>
<ul>
<li><p>Associated biological and cellular processes (Gene Ontology terms)</p></li>
<li><p>Where and when it is expressed in the body</p></li>
<li><p>Reported variants and associated diseases</p></li>
<li><p>Known post-translational modifications and splice variants</p></li>
<li><p>Known interacting proteins</p></li>
<li><p>Structures (from the PDB, EMDB, AlphaFold models)</p></li>
<li><p>Domains and motifs</p></li>
<li><p>The sequence of the protein</p></li>
</ul>
<p>NCBI has similar information curated in a slightly different way. It is
easy to get sequences from different species from NCBI. In addition to
the information in UniProt, there is also information about small
molecules that bind to your molecule of interest, links to Pubmed, and
information from RNAseq datasets. Genomic sequences are labeled with the
prefix NG, RNA/cDNA sequences are labeled NM, and proteins are labeled
NP. If you look at the NP file, it is a text file that includes links to
the sequence, Pubmed references, information about PTMs, variants, and
functional domains. There are also direct links to BLAST, structures,
conserved domains, and more!</p>
<blockquote>
<p><strong>Pro-tip:</strong> Record the exact reference sequence identifier that you are using for your studies and report it in your paper. This will prevent
confusion between splice variants/related genes. When looking at other
publications, it is important to note that they might be using a
different reference sequence. It is not uncommon to find the same
mutation reported in the literature with different numbering because
people are using a different reference sequence. Always double check
this before starting your experiments. Also, if you are working with a
plasmid, sequence it and double check that it matches the reference
sequence you think it should.</p>
</blockquote>
<p><u></u></p>
<p><u>Conservation across evolution</u></p>
<p>Over the course of evolution, random genetic mutations have contributed
to the development of species with different genomes, transcriptomes,
and proteomes. These random mutations have led to sampling of a vast
sequence space. Some genetic mutations are tolerated or harmless, while
others impact fitness (both positively and negatively). As such, there
are certain sequences that are more highly conserved due to evolutionary
pressure, while other stretches are more variable.</p>
<p>Looking at sequence conservation can help you to form educated
hypotheses about molecules. In general, if there is a strong
evolutionary pressure for maintaining a particular residue or sequence,
then this sequence will be more highly conserved. Different things can
be conserved. For example, in the context of a protein, a sequence can
be absolutely conserved. Alternatively, there might be conservative
mutations, where the sequence changes but the charge or hydrophobicity
is conserved (e.g., changing glutamine to aspartic acid preserves the
charge). This is frequently seen with residues involved in forming salt
bridges, where maintaining the charge preserves this interaction, but
charge reversals or neutralization are damaging. Moreover, within
disordered regions, there is often more variation of sequence, but
physical properties are conserved (e.g., the charge patterning is
conserved).</p>
<p>It is also sometimes useful to look at different, but related molecules
when forming hypotheses. For example, if there are several related
isoforms of a given protein, it is often useful to look at where these
differences occur. This analysis can help you to form hypotheses related
to what residues are absolutely necessary for function and what residues
are responsible for differences between isoforms. For example, some
related isoforms may have different binding partners, and these
differences could arise from sequence variation and/or insertion of
additional domains. Alternatively, small differences at an active site
could potentially explain differences in catalytic activity. Finally, it
is sometimes useful to broadly look for conservation across different
proteins. For example, two different molecules with ATPase activity
might show some degree of conservation near the active site, and this
can sometimes be used to identify important domains within proteins that
haven’t been studied before.</p>
<p>It is important to note that it is difficult to assess conservation in
closely related species since by definition, they are more genetically
similar than distantly related species. For example, there are some
sequences that are highly conserved within a certain evolutionary clade,
but this sequence may look quite different in a different clade. Thus,
when evaluating sequence conservation, it is important to look at
multiple, distantly related species.</p>
<p>A common approach for looking at conservation is to collect sequences
from multiple, divergent species. Both NCBI and UniProt have collections
of sequences from different species. Often, it is useful to get several
vertebrates (e.g., human, mouse, rat, pig, zebrafish) and some
invertebrates (e.g., C. elegans, drosophila). There are multiple formats
that can be used for downloaded sequences, but one of the most commonly
used ones is FASTA. Multiple sequences can then be aligned in a program
such as Clustal Omega, and one can then examine which residues are
conserved. It is often helpful to color code the sequences by the type
of residue (e.g., aromatic, charged, polar, nonpolar). It is also
important to note your reference sequence, since the numbering of amino
acids can vary from species to species.</p>
<p>If you are trying to identify other proteins that share common features
with your protein of interest. One way of doing this is to use BLAST.
BLAST allows you to input any sequence and then search for similarities
with any other sequence, regardless of whether it is the same protein or
not. For example, if I wanted to identify human proteins with
similarities to human ventricular cardiac myosin (MYH7), I could use
BLAST the MYH7 sequence against other human proteins. In this example,
the search will return multiple other myosin isoforms. Alternatively, if
you have a protein with a specific domain that you are not sure whether
it is found elsewhere in the proteome, you could blast the sequence to
see what other proteins share sequence similarity with your protein of
interest.</p>
<blockquote>
<p><strong>Pro tip:</strong> One particularly useful feature of BLAST is that you can
use it on unknown DNA sequences. For example, suppose that you find a
mystery plasmid in the freezer, or a collaborator sends you a plasmid.
The first step is to send the plasmid out for sequencing. Once the
sequencing is returned, you can use BLAST to check what DNA is in the
plasmid. Moreover, if you introduced a mutation into a plasmid, you
could run the sequence of the mutated plasmid into BLAST to make sure
that you have only mutated your nucleotide of interest.</p>
</blockquote>
<p><u></u></p>
<p><u>Single nucleotide polymorphisms/variants</u></p>
<p>There is not a single wild-type human genome. Every human is different,
and every human genome is different. While there are certainly some
variants associated with disease (both monogenic and polygenic risk
factors), there are also a lot of variants that are benign (e.g.,
variants in genes that determine eye color or whether cilantro tastes
like soap). The most common genetic changes are single nucleotide
variations (SNVs) where one nucleotide is substituted for another.
Occasionally, you will see a distinction between single nucleotide
polymorphisms (SNPs) which occur in a sizeable portion of the population
and SNVs which occur in less than 1% of the population, but here, we
will use these terms interchangeably for simplicity.</p>
<p>It is important to note that some SNPs/SNVs are enriched in certain
subpopulations, and this information is used by companies to predict
ancestry. There are multiple cases in the literature where specific
variants have been incorrectly associated with disease because the
variant was compared to only a single reference sequence. As such, when
looking at variants, it is important to ensure that your database
includes populations drawn from all over the world.</p>
<p>One place to look at SNPs and SNVs is the <u>gnomAD database</u>.
This is an online database that includes exomes and genomes from
&gt;750,000 “healthy” people from around the world. You can look at your
gene of interest and look at all of the SNVs in your gene, along with
the frequency of that SNV in the population. The database includes
synonymous mutations (i.e., the nucleotide changes but the protein
sequence does not), missense variants (i.e., the nucleotide changes,
leading to a change in protein sequence), indels (i.e., a nucleotide is
inserted or deleted, changing the protein sequence), and mutations in
introns. Importantly, you can also look to see whether a given variant
is potentially associated with disease.</p>
<p>gnomAD also provides several other useful insights. gnomAD calculates
the number of variants that you would expect to see in your gene of
interest given normal genetic variation. This is then compared to the
number of observed variants. If you observe fewer variants than
expected, this typically means that this sequence is constrained, and
variants within this sequence are not well tolerated. You can look at
the o/e ratio, which is the ratio of observed/expected. A
<u>smaller</u> o/e ratio means that the sequence is more
constrained. Similarly, you can look at whether the gene is essential by
looking at the ratio of loss of function variants (if no loss of
function variants are seen, this suggests an essential gene). Another
useful metric given by gnomAD is the Z score (genomic constraint metric)
such that a more constrained sequence will have a <u>higher</u> Z
score. You can also look at the sequence to see whether pathogenic/loss
of function variants are enriched in a particular part of the sequence.
This information can then be cross-referenced to map these on to
specific functional domains of your protein.</p>
<p>If you are looking for variants in your molecule of interest potentially
associated with disease, there are multiple places to look for
information including OMIM and ClinVar. ClinVar is a curated list of
variants that have been found in patients. You can search for your
molecule of interest and get a list of variants found in the clinic.
Variants are classified as benign, likely benign, variants of unknown
significance (VUS), likely pathogenic, or pathogenic based on the
strength of the clinical data associated with the variant.</p>
<p>As a note of caution, it can be difficult to unambiguously determine
whether a given variant is causative of disease. Some variants are
labeled as likely pathogenic in ClinVar, but then when you look in
gnomAD (“healthy” patients), you see that there are a lot of healthy
people with that specific variant, suggesting that it might not be a
monogenic cause of disease. Moreover, there are some variants labeled as
pathogenic in ClinVar based on a single reporter. (There are also papers
based on a single patient without additional evidence). As a rule of
thumb, a truly pathogenic variant will be rare in the gnomAD database,
it will have been found in multiple unrelated family members, and
pedigree analysis will show segregation of the variant with the disease.</p>
<p><u>Transcriptomics</u></p>
<p>Gene expression varies from cell to cell, and the exact transcriptome
varies with development and disease. Moreover, even within a given cell
type, gene expression can vary depending on the physiological state of
the cell. For example, fibroblasts can undergo activation to
myofibroblasts by TGFB activation or mechanical stretch, leading to
major changes in cellular physiology and gene expression. There are
large, curated databases of transcriptomic data, including the GEO
Database. These databases include bulk RNAseq, single cell sequencing,
and single nuclei sequencing.</p>
<p><u></u></p>
<p><u>Proteomics and post-translational modifications (PTMs)</u></p>
<p>The proteome varies from cell to cell across development and disease.
Moreover, there are changes in the isoforms of proteins that are
expressed associated with different cellular states. For example, cancer
cells often show dramatic changes in their proteomes. Changes in gene
expression are often correlated with changes in protein expression, but
this is not always the case. Advances in mass spectrometry have enabled
the examination of whole proteomes (i.e., the most abundant proteins,
but not necessarily all proteins), albeit with lower sensitivity and
depth than RNAseq. There are multiple databases out there consisting of
whole proteomes, including UniProt and ProteomeCentral.</p>
<p>Many proteins are processed and post-translationally modified. The
number of identified PTMs is constantly growing. Moreover, there are a
large number of PTMs associated with development, health, and disease.
Many PTMs are associated with changes in protein structure and function.
For example, phosphorylation can activate and inactivate many proteins.
Ubiquitination can mark proteins for degradation. Proteins such as
microtubules are heavily modified to interact with different binding
partners. There are multiple databases out there consisting of
information about PTMs, including the MassIVE database and PhosphoGRID.
Care should be taken to focus on PTMs that have been validated in
multiple experiments.</p>
</div>
<div id="in-class-activity" class="section level2 hasAnchor" number="1.2">
<h2><span class="header-section-number">1.2</span> In-class activity<a href="informatics.html#in-class-activity" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><strong><u>In Class Activity 1: Harnessing public repositories and
“Omics”</u></strong></p>
<p><strong><u>Section 1: Basic Information:</u></strong></p>
<p>For this section, we will use the protein calmodulin as a model (CALM1).</p>
<ol style="list-style-type: decimal">
<li><p>Look up human calmodulin 1 (CALM1) in UniProt.</p>
<ol style="list-style-type: lower-alpha">
<li><p>What is the UniProt ID number?</p></li>
<li><p>List 4 functions of CALM1 and provide references for these
functions.</p></li>
<li><p>What are 2 diseases caused by mutations in CALM1? Provide basic
background information on these diseases. For each disease, list
1 variant associated with disease and provide a reference for
this variant.</p></li>
<li><p>Provide a list of 3 interacting partners.</p></li>
<li><p>Look at the structure of the protein. Several structures will
come up. Look at PDB 1CLL. Snap a screenshot of the structure.
Provide the AlphaFoldDB ID number.</p></li>
<li><p>The protein has 4 well defined domains. What are these domains
called? Where in the structure are these domains? What does this
domain do?</p></li>
<li><p>Find the sequence of CALM1. Copy and paste the sequence here.
How many amino acids are in the structure? What is the molecular
weight of the protein?</p></li>
</ol></li>
<li><p>Go to NCBI and look up CALM1.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Click on GENE: CALM1</p>
<ol style="list-style-type: lower-roman">
<li><p>Go to the bibliography. Provide several references about
CALM1.</p></li>
<li><p>What are 2 diseases caused by CALM1? Click on the
GeneReviews links for these diseases. Copy information about
the clinical characteristics and management?</p></li>
<li><p>Return to the GENE:CALM1 page. List several functions and
processes associated with CALM1.</p></li>
<li><p>Go to the section, “mRNA and Protein(s)”. There are several
isoforms listed. Look at your UniProt accession number in
Part 1. Which isoform matches this UniProt ID.</p>
<ol style="list-style-type: decimal">
<li><p>The mRNA is listed with the prefix NM_. Click on this
and scroll down to see the sequence.</p></li>
<li><p>Go back. What is the accession number for the protein
(NP_)? Click on the link.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Provide several references for this protein.</p></li>
<li><p>List 3 sites of post-translational modifications.
Scroll all the way down to see the sequence.</p></li>
<li><p>Scroll to the top of the page. You download the
sequence in several formats (there is a blue link
with the word GenPept). Click on FASTA. This is a
common file format. Copy and paste the FASTA
sequence into your word document.</p></li>
</ol></li>
</ol></li>
</ol></li>
</ol></li>
</ol>
<p><strong><u>Section 2: Sequence Alignment and Conservation</u></strong>:</p>
<p>For this section, we will use cardiac troponin T (TNNT2) as a model</p>
<ol style="list-style-type: decimal">
<li><p>Go to the NCBI page and search for TPM1.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Click on Proteins to go to the protein section.</p></li>
<li><p>Click on otrhologs. This will bring up a list of CALM1 in other
species. Click the check box next to at least 10 species. Try to
get a diverse array of species.</p>
<ol style="list-style-type: lower-roman">
<li><p>Click “download”, select “Refseq Proteins FASTA” and “one
protein per gene”.</p></li>
<li><p>This will save a FASTA file on the computer. It should open
with a text editor program. Open the file and copy the text.</p></li>
</ol></li>
</ol></li>
<li><p>Open Clustal Omega (use Google). This is the alignment program.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Paste your sequences into the search box. Then click submit. (If
you do this with your own data in the future, you can give the
job a name to retrieve it later).</p></li>
<li><p>Click view results.</p></li>
<li><p>Click the “Tool Output” tab.</p>
<ol style="list-style-type: lower-roman">
<li><p>Click download to download a text file with the alignment.
This should open with a text editor.</p></li>
<li><p>Scroll down. You should see “alignment with colors”. Click
this button. This shows each sequence. * denotes completely
conserved. Residues are colored by charge and
hydrophobicity.</p></li>
</ol></li>
<li><p>Scroll back up. Click the “Alignments” tab.</p>
<ol style="list-style-type: lower-roman">
<li>Give it a second. The resolution improves. Drag the mouse to
look through the sequence.</li>
</ol></li>
<li><p>Scroll up again. Click the “Results files” tab.</p>
<ol style="list-style-type: lower-roman">
<li>You can now pick which outputs you want to save.</li>
</ol></li>
</ol></li>
</ol>
<p><strong><u>Section 3: Looking at Variants</u></strong>:</p>
<p>For this section, we will use tropomyosin (TPM1) as a model</p>
<ol style="list-style-type: decimal">
<li><p>Go to the gnomAD database. Enter TPM1 as your protein of interest.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Look at the top right corner of the screen.</p>
<ol style="list-style-type: lower-roman">
<li><p>Are synonymous mutations likely damaging? Why?</p></li>
<li><p>Are missense mutations likely damaging? Why?</p></li>
</ol></li>
<li><p>Scroll down to the long table. Click on “Germline
classification” to sort by clinical classification from ClinVar.</p>
<ol style="list-style-type: lower-roman">
<li><p>Look at the HGVS consensus to see the variant consequences.
Look at VEP annotation to see the type of change.</p></li>
<li><p>What are 3 pathogenic/likely pathogenic variants?</p></li>
<li><p>What are 3 benign/likely benign variants?</p></li>
<li><p>What are 3 variants of unknown significance?</p></li>
<li><p>Find a mutation with conflicting interpretation with a large
number of allele counts in the population. Is this likely
pathogenic? Why or why not?</p></li>
</ol></li>
</ol></li>
<li><p>Open ClinVar. Enter TPM1 as your protein of interest.</p>
<ol style="list-style-type: lower-alpha">
<li><p>On the left side of the screen, you can see filters.</p>
<ol style="list-style-type: lower-roman">
<li><p>Click “pathogenic”. What diseases are associated with
mutations in TPM1?</p></li>
<li><p>Find E62K. Click on this entry.</p>
<ol style="list-style-type: decimal">
<li>Scroll down. How many submitters are there? What
evidence do they present?</li>
</ol></li>
<li><p>Go back. Find V95A. Click on this entry.</p>
<ol style="list-style-type: decimal">
<li><p>How many submitters are there for this mutation?</p></li>
<li><p>List 2 publications associated with this mutation.</p></li>
</ol></li>
</ol></li>
</ol></li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/01-informatics.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
